packages/next/src/build/webpack-config.ts

```diff
@@ -473,6 +473,47 @@ export default async function getBaseWebpackConfig(
       )
     }
 
+    const useBuiltinSwcLoader = process.env.BUILTIN_SWC_LOADER
+    if (isRspack && useBuiltinSwcLoader) {
+      return {
+        loader: 'builtin:next-swc-loader',
+        options: {
+          isServer: isNodeOrEdgeCompilation,
+          rootDir: dir,
+          pagesDir,
+          appDir,
+          hasReactRefresh: dev && isClient,
+          // nextConfig: config,
+          // jsConfig,
```

Comment:
Any idea why these are commented out?

Reply:
Looks like they're needed in the alternate branch but not when running with rspack's built-in loader? @SyMind do you know why this is the case?